### ISSUE TITLE
When will the official launch of this will be done.

### ISSUE THREAD

**Issue by keerthinath-ganjam** [Microsoft/ApplicationInsights-SDK-Labs#123]
When will the official launch of this will be done.

**Comment by keerthinath-ganjam** [comment]
Please respond

**Comment by tomasr** [comment]
We've just published an updated version of this SDK. Status has not changed at the moment. @SergeyKanzhelev do you want to add anything here?

**Comment by SergeyKanzhelev** [comment]
@keerthinath-ganjam what is the main request here? Have it on nuget.org or have an official support statement from Microsoft?

**Comment by Hamaze** [comment]
The request is definitely to have it on nuget.org. It's a big pain in the rear for teams and build systems not having it on nuget.org.

**Comment by jiberan** [comment]
related to #5

**Comment by keerthinath-ganjam** [comment]
Yes main request is to have it on the nuget.org.

**Comment by keerthinath-ganjam** [comment]
Any plans of releasing it to nuget.org, if so, by when.

**Comment by haldiggs** [comment]
nuget would be great. Its a real pain to force my code build systems to grab this at the moment.

**Comment by keerthinath-ganjam** [comment]
@SergeyKanzhelev , @tomasr : Any updates??

**Comment by keerthinath-ganjam** [comment]
Is any activity happening in this repo ???

**Comment by ScottRapsey** [comment]
Is this project dead or is it done?
If it’s dead, did the functionality get rolled into the regular Application Insights SDK?
If it’s done, why is it still in Pre Release and why is it on MyGet instead of Nuget?
Just looking for some clarification as to whether I should, or even need to, be using this in my IIS hosted WCF applications.

**Comment by bearer-syntaxerror** [comment]
My team is currently using this library in production. If it remains on myget, it's only a matter of time before the developers get frustrated and remove it from our code base. We'd like it rolled into the official application insights nuget package or to be hosted on nuget.org if it continues to be a standalone package. If neither will be done, please let us know.

**Comment by bearer-syntaxerror** [comment]
Until this package is posted to nuget, you can keep your team in sync by adding a NuGet.config file to your project containing the following:
```
<packageSources>
   <add key="applicationinsights-sdk-labs" value="https://www.myget.org/F/applicationinsights-sdk-labs" />
</packageSources>
```
per [https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#packagesources](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#packagesources)

**Comment by haldiggs** [comment]
That hells a little. However I still have to run "special" code scans and
repeatedly justify this for external auditors. Kinda sucks since this
library is the only practical way to monitor WCF in Azure... That is what I
have seen... If there are better feel free to say it

**Comment by bearer-syntaxerror** [comment]
I know of nothing better. The standard Application Insights package from nuget isn't nearly as robust as this one. Moving the ApplicationInsights-SDK-Labs Application Insights WCF package official and hosting it in nuget's official repository **would be a huge value-add** for those of us still using .net WCF. Thank you.

**Comment by haldiggs** [comment]
> I know of nothing better. The standard Application Insights package from nuget isn't nearly as robust as this one. Moving the ApplicationInsights-SDK-Labs Application Insights WCF package official and hosting it in nuget's official repository would be a huge value-add for those of us still using .net WCF. Thank you.

I agree... not sure why nobody is monitoring this from Microsoft. I need this... I am tired of having to justify it usage during code audits. I need this in a standard package please sirs.

**Comment by dnelly** [comment]
Same here.
